Description: Remove stale Cython source from PyYAML
 PyYAML generates C source during the build but the source distribution
 included in GPDB is from PyPI and includes stale source. Since certain headers
 have been moved between Python 3.9 and Python 3.10, this source no longer
 compiles. Removing it should force it to be re-generated when running
 setup.py.
Author: Bradford D. Boyle <bradford.d.boyle@gmail.com> 
Forwarded: no
Last-Update: 2023-02-04

--- a/gpMgmt/bin/Makefile
+++ b/gpMgmt/bin/Makefile
@@ -103,6 +103,7 @@
 pyyaml:
 	@echo "--- pyyaml"
 	cd $(PYLIB_SRC_EXT)/ && $(TAR) xzf $(PYYAML_DIR).tar.gz
+	rm -f $(PYLIB_SRC_EXT)/$(PYYAML_DIR)/ext/_yaml.c
 	cd $(PYLIB_SRC_EXT)/$(PYYAML_DIR)/ && env -u CC $(PYTHON) setup.py build
 	cp -r $(PYLIB_SRC_EXT)/$(PYYAML_DIR)/build/lib*-3*/* $(PYLIB_DIR)/
 
